lib/src/memcache_native_connection.dart

```diff
@@ -757,7 +757,7 @@ class MemCacheNativeConnection {
   }
 
   void onError(error) {
-    close(error);
+    close(error).catchError((_) {});
```

Comment:
Can you move this instead to `await socket.close()` further down, like:

```dart
  try {
    await socket.close();
  } catch (_) {}
```

Reply:
Sure! I'd put it here to avoid hiding errors if `close()` was called directly, but since we're failing the pending requests anyway, we might as well catch the errors there.